### DISCUSSION TITLE
Using Shadow Command

### DISCUSSION THREAD

**Discussion by mwgevans** [dlemstra/Magick.NET#1307] [Help]
Hi, I'm trying to use the Shadow command, but am struggling to find any examples. I have the following in powershell.
`$arguments = @(
    'convert',
    "wm_toes_wm.png",
    '-bordercolor', 'white',
    '-border', 1, '-bordercolor', 'black', '-border', 1,
    '-background', 'black', '(', '+clone', '-shadow', '80x5+10%', ')', '+swap', '-background', 'white', '-flatten', '-bordercolor', 'White', '-border', 'x20%+100',
    'test.png'
)
& magick.exe $arguments`

I have tried
`                image.BorderColor = new MagickColor("White");
                image.Border(1);
                image.BorderColor = new MagickColor("Black");
                image.Border(1);
                image.BackgroundColor = new MagickColor("Black");
                var image1 = image.Clone();
                image1.Shadow(50, 50, 0.8, (Percentage)10);
                collection.Add(image);
                collection.Add(image1);
                var newimage = collection.Flatten(new MagickColor("white"));
                newimage.BorderColor = new MagickColor("White");
                newimage.Border(25);
                newimage.Write("test.jpg");
`
with various different parameters on the shadow command, but I am at a loss!

**Comment by dlemstra** [comment]
Can you minimize this code sample to only your issue with Shadow? And I think it should be `image1.Shadow(80, 5, 0.8, (Percentage)10);` ?

**Reply by mwgevans** [reply]
I believe that is the root of the problem, but my knowledge is weak!
The original line was `shadow', '80x5+10%'`
